packages/xrpl/src/models/transactions/paymentChannelClaim.ts

```diff
@@ -147,12 +153,20 @@ export function validatePaymentChannelClaim(tx: Record<string, unknown>): void {
     throw new ValidationError('PaymentChannelClaim: Channel must be a string')
   }
 
-  if (tx.Balance !== undefined && typeof tx.Balance !== 'string') {
-    throw new ValidationError('PaymentChannelClaim: Balance must be a string')
+  if (
+    tx.Balance !== undefined &&
+    typeof tx.Balance !== 'string' &&
```

Comment:
This clause needs to be removed - if it's an amount this will fail.

Reply:
It can be an amount or string I believe.